scripts/Compile-Zstddec.ps1

```diff
@@ -0,0 +1,14 @@
+# This PowerShell script (principally for Windows 10/11) compiles zstddec.js using docker according to the values set in 
+# /emscripten/ztandard/compile.sh . Please be sure to review the commandline in compile.sh before running this script, to
+# be sure it's doing what you want.
+#
+# Prerequisites:
+#
+#     * Ensure WSL 2 is enabled in Windows (WSL = Windows Subsystem for Linux) 
+#     * Install docker desktop on Windows: choose the WSL 2 docker backend (not the standard/legacy one)
+#     * Open docker and ensure the docker daemon is running correctly (icon should be green)
+#     * Ensure docker desktop is signed in to your dockerhub account
```

Comment:
It's probably not necessary, except that downloading from the docker hub will be faster if signed in

Reply:
I tried it without, but it fails saying it needs username and password. Seems like that one doesn't allow anonymous download. Rather than providing name/pwd in the script (as they would be personal) I thought it best just to ask the user on Windows to ensure docker desktop is logged in. May be different on linux.